### PR TITLE
feat: add Node Lambda

### PR DESCRIPTION
**Pull request by schuettc**
**Issue #:**

Add PSTN Audio SIP media application handler for Serverless Application Repository

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+samconfig.toml
+packaged-template.yaml
+.aws-sam
```
Comment (xuesichao) [inline utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/.gitignore:3]: nit: new line end of file

File: utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/template.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      Architectures:
+        - arm64
+
+Outputs:
+  PSTNAudioHandler:
+    Description: 'PSTNAudioHandler Lambda Function ARN'
+    Value: !GetAtt PSTNAudioHandler.Arn
+  PSTNAudioHandlerIamRole:
+    Description: 'Implicit IAM Role created for PSTNAudioHandler function'
+    Value: !GetAtt PSTNAudioHandlerRole.Arn
```
Comment (simmkyu) [inline utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/template.yaml:29]: Just to confirm. Is `PSTNAudioHandlerRole` automatically generated for the `PSTNAudioHandler` resource?

Comment (schuettc) [reply utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/template.yaml:29]: That's correct.  It's just part of the SAM template and generated by SAM.

File: utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/app.js [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+};
+
+const resumeCallRecordingAction = {
+  // https://docs.aws.amazon.com/chime/latest/dg/resume-call-recording.html
+  Type: 'ResumeCallRecording',
+  Parameters: {
+    CallId: '', //required - Allowed values – A valid call ID
+  },
+};
+function callIdToHangup(participants) {
```
[the remark this answers is not in the document]
Comment (schuettc) [reply utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/app.js]: Updated with this.

File: utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/app.js [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+  var hangupId = participants.filter(function (participant) {
```
[the remark this answers is not in the document]
Comment (schuettc) [reply utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/app.js]: Removed in favor of alternative.

**Pull request merged**